### PR TITLE
Overall interface review

### PR DESCRIPTION
**Pull request by brunosabot**
Combined work with @cecilefreyd
We made an overall interface review to have light aggressive colors than before, anf make the design smoother.
The work includes:
 - Light navbar with red text, smooth hover,
 - Responsive lightbar,
 - Moving the loader to a tinier space to avoid the feeling of floating elements,
 - Complete review of the stats screen,
 - Global title and subtitle style adjustments.

Here is some screenshots to see what the app now looks (and I hope you will like):

screen shot 2018-12-21 at 6 15 42 pm
screen shot 2018-12-21 at 6 14 47 pm
screen shot 2018-12-21 at 6 14 55 pm
screen shot 2018-12-21 at 6 15 13 pm
screen shot 2018-12-21 at 6 15 30 pm

### CODE REVIEW

**Comment by hgwood** [comment]
Thank you so much! It looks super nice!

@Errorname what would you think of back-porting this to FAQ?

**Comment by Errorname** [comment]
It would be awesome to have a design system/library for Zenika tools. I'm looking forward to see how we could backport this to FAQ.

@brunosabot If I can help and give feedback, don't hesitate to contact me!

**Comment by Bugsyaya** [comment]
At first, really nice !

I like the nav bar because all informations are clear. We really need to have the github button when we are on mobile ?

I think the screens ("we're closed" and "hi!") are really sad. It looks like pop-ups.

May be we can make some thing for the stats table ? I find it cumbersome and informations inside not readable. Furthermore, may be change the green because at first sight, I think it's black. Perhaps we can change the dropdown "global" by "tab". Like this, the user can see all of choice. Finally, it's may be better if emoji are bigger.

(désolée pour mon anglais... j'espère que tout le monde comprendra ^^')

**Comment by Ked57** [comment]
global means you're viewing stats for all zenika agencies

**Comment by Ked57** [comment]
@brunosabot i'll have a look at it this afternoon. Did you create a repo for the navbar ? We put the the github logo in a file here: https://github.com/Zenika/billing-platon/blob/master/ui/public/github.svg
Right now it's hardcoded in a html tag which is not really good

**Comment by Ked57** [comment]
@Bugsyaya maybe we could use one of the robots you made for billing-platon for the "we're closed" page ?

**Comment by hgwood** [comment]
Let's be careful not to expand the scope of this PR. Of course there are lots of stuff that can be improved further but it does not have to block this PR. I feel all comments are related to issues that are already present in the app, thus unrelated to the improvements proposed by this PR. What do you think?

TLDR: let's merge it. @brunosabot @cecilefreyd is that OK for you?

**Comment by Ked57** [comment]
We revert the addition of tabs for another PR, and we will merge the new UI now
